packages/cli/__tests__/zosfiles/__system__/compare/ds/Compare.ds.system.test.ts

```diff
@@ -0,0 +1,77 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { Session } from "@zowe/imperative";
+import { Create, Delete, CreateDataSetTypeEnum } from "@zowe/cli";
+import * as path from "path";
+import {ITestEnvironment, runCliScript} from "@zowe/cli-test-utils";
+import {TestEnvironment} from "../../../../../../../__tests__/__src__/environment/TestEnvironment";
+import {ITestPropertiesSchema} from "../../../../../../../__tests__/__src__/properties/ITestPropertiesSchema";
+import { getUniqueDatasetName} from "../../../../../../../__tests__/__src__/TestUtils";
+
+let REAL_SESSION: Session;
+let testEnvironment: ITestEnvironment<ITestPropertiesSchema>;
+let defaultSystem: ITestPropertiesSchema;
+let dsname: string;
+const TIMEOUT = 20000;
+
+
+describe("View Data Set", () => {
```

Comment:
The `view data-set` command is mentioned several times throughout this file in the test and script names. Is this intentional, or should it be referring to `compare data-set`?